### PR TITLE
public: book service

### PR DESCRIPTION
**Pull request by Valentin-Stratan**

### CODE REVIEW

File: backend/utils/booking-utils.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const mailFormat = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
```
Comment (ruben-ilciuc) [inline backend/utils/booking-utils.js]: Import it from utils.

File: backend/utils/booking-utils.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+module.exports = {
+    validateEmail: function validateEmail(email) {
```
Comment (ruben-ilciuc) [inline backend/utils/booking-utils.js:3]: Try to simplify this function into a single line of code.

File: backend/utils/booking-utils.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            return false;
+        else {
+            let isValid = mailFormat.test(email);
+            if (!isValid)
+                return false;
+            else
+                return true;
+        }
+    },
+    validatePhone: function validatePhone(phone) {
```
Comment (ruben-ilciuc) [inline backend/utils/booking-utils.js:6]: Also, simplify this one.

File: backend/utils/booking-utils.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        if(!phone)
+            return false;
+        else {
+            if(phone.length > 16)
+                return false;
+            else 
+                return true;
+        }
+    },
+    validateName: function validateName(name) {
```
Comment (ruben-ilciuc) [inline backend/utils/booking-utils.js:9]: Also, simplify this one.

**Review by ruben-ilciuc — CHANGES_REQUESTED** [review]

File: backend/utils/booking-utils.js (hunk 1 above)
Comment (Valentin-Stratan) [reply backend/utils/booking-utils.js]: I separated the utils file according to controllers, booking-utils/company-utils/service-utils

Comment (ruben-ilciuc) [reply backend/utils/booking-utils.js]: Right, but the principle of DRY (Don't Repeat Yourself) is broken since there is a copy of it in the utils file.

**Review by ruben-ilciuc — APPROVED** [review]
:+1:

**Pull request merged**